test/Tests.Infrastructure/RavenTestBase.Cluster.cs

```diff
@@ -240,5 +241,50 @@ public void SuspendObserver(RavenServer server)
 
             server.ServerStore.Observer.Suspended = true;
         }
+
+        public void AssertNumberOfCommandsPerNode(long expectedNumberOfCommands, List<RavenServer> servers, string commandType, int timeout = 30_000, int interval = 1_000)
+        {
+            var numberOfCommandsPerNode = new Dictionary<string, long>();
+            var isExpectedNumberOfCommandsPerNode = WaitForValue(() =>
```

Comment:
let's use `WaitForValueAsync` instead, the 1s interval is too much here for doing a `Thread.Sleep`